src/main/java/seedu/address/logic/parser/AddressBookParser.java

```diff
@@ -6,15 +6,7 @@
 import java.util.regex.Matcher;
 import java.util.regex.Pattern;
 
-import seedu.address.logic.commands.AddContactCommand;
-import seedu.address.logic.commands.ClearContactCommand;
-import seedu.address.logic.commands.Command;
-import seedu.address.logic.commands.DeleteContactCommand;
-import seedu.address.logic.commands.EditContactCommand;
-import seedu.address.logic.commands.ExitCommand;
-import seedu.address.logic.commands.FindContactCommand;
-import seedu.address.logic.commands.HelpCommand;
-import seedu.address.logic.commands.ListContactCommand;
+import seedu.address.logic.commands.*;
```

Comment:
Consider importing only necessary classes instead of the whole package. This will make sure that it passes the checkstyle!